source/lambda/apiprocessor/lambda_function.py

```diff
@@ -85,6 +83,24 @@ def lambda_handler(event, context):
                     request["documentId"] = event['queryStringParameters']['documentId']
                 result = search(request)
 
+        # a specific document redaction items (terms, headers and footers)
```

Comment:
Not specifying `documentId` for the `redaction` APIs causes 500 error

Reply:
is this a redaction feature or search related issue?

Reply:
The front end will always pass a `documentId` as expected but the API should never 500 from a foreseen cause rather than gracefully handling the misshapen request